shared/oae/macros/list.html

```diff
@@ -179,17 +181,23 @@
             <!-- COMPACT LIST VIEW -->
             <div class="oae-list-compact-item">
                 <div class="oae-listitem">
-                    ${renderCheckbox(entityData, 'compact', displayOptions.showCheckbox)}
-                    <div class="oae-listitem-primary-thumbnail">
-                        ${renderThumbnail(entityData, displayOptions)}
-                    </div>
-                    {if displayOptions.listItemActions}
-                        <div class="oae-listitem-actions">
-                            ${displayOptions.listItemActions()}
+                    <div class="oae-listitem-inner">
```

Comment:
The introduction of the `inner` class should be documented in the list item component documentation